### DISCUSSION TITLE
Zoom Recordings/Access to Past Lectures

### DISCUSSION THREAD

**Discussion by nkacharia** [CS342/.github#11] [Setup] [answered]
### Description

When I click on the zoom tab in CS 342, there is a warning/error message. I have tried launching zoom again and consulted a friend who is also in the course, she cannot access the recoding either. 

![Snip 2023-01-22 21 53 56](https://user-images.githubusercontent.com/81988289/213975405-85947516-97bc-463a-8012-a2a8784d2d32.png)

### Reproduction

1. Go to canvas
2. Click the course
3. Click the zoom tab
4. There is a warning sign, but click the zoom link anyways

### Expected behavior

Expect to be able to access previous lectures in Zoom's previous meeting tabs 

### Additional context

Need to access this past Thursday's Lecture to be able to do my homework. Thanks :)

### Code of Conduct

- [X] I agree to follow this course's Code of Conduct and Contributing Guidelines

**Comment by vishnuravi** [comment, marked as answer]
Hi @nkacharia,

This appears to be an issue with Stanford Zoom - we will file a bug report with IT. Hopefully it will be fixed soon. In the meantime, let us know if we can answer any questions about the homework for you!

Thanks,
Vishnu

**Reply by vishnuravi** [reply]
@nkacharia I just checked and the Zoom tab seems to be working again. Take a look and let me know if you're able to get to the recordings. Thanks!

**Comment by aalami5** [comment]
Stanford did send an email last week about issues. Looks good on my end. Be sure to go to "cloud recordings" to see recordings.
